src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -332,13 +335,71 @@ public override IntermediateNode Visit(TexlCallNode node, IRTranslatorContext co
                     }
                 }
 
+                args = ReplaceBlankWithDefault(args, func);
```

Comment:
Is the intention that this become a list of implicit pre-coercions by type, or should there be a helper function?